### PR TITLE
Merge development into master

### PR DESCRIPTION
**Pull request by jakemulley**

### CODE REVIEW

File: src/styles/modules/email-form.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.email-form {
+  display: flex;
+  margin-bottom: $base-spacing-unit-small;
+  color: $grey-1;
+  input[type=email] {
+    -webkit-appearance: none;
+    border-radius: 0;
+    border-right: 0;
```
Comment (UsmanAfzal) [inline src/styles/modules/email-form.scss]: Duplicate border-right property

File: src/styles/modules/email-form.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    border-bottom-left-radius: 4px;
+    border-top-left-radius: 4px;
```
Comment (UsmanAfzal) [inline src/styles/modules/email-form.scss]: Can shorthand be used?

File: src/styles/modules/email-form.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    border: 1px solid $grey-3;
+    border-right: 0;
+    padding: $base-spacing-unit $base-spacing-unit-tiny;
+    width: 100%;
+  }
+  .button {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
```
Comment (UsmanAfzal) [inline src/styles/modules/email-form.scss:15]: Can shorthand be used?

**Review by UsmanAfzal — CHANGES_REQUESTED** [review]
Comments added

File: src/snippets/email-signup.liquid [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{% comment %} This code is specific to MailChimp - please do not change {% endcomment %}
+<form action="https://parliament.us16.list-manage.com/subscribe/post?u=64af15924ba58f178f281a7e7&amp;id=15aaa277c6" method="post" target="_blank" class="email-form">
+  <h3 class="visually-hidden">{{ 'email_signup.subscribe_to_our_list' | t }}</h3>
+  <label for="mce-EMAIL" class="label-hidden">{{ 'email_signup.email' | t }}</label>
+  <input id="mce-EMAIL" name="EMAIL" type="email" placeholder="Email Address" value="">
+  <div aria-hidden="true" style="position: absolute; left: -5000px;">
```
Comment (naseberry) [inline src/snippets/email-signup.liquid:6]: Why do we have inline css?

Comment (jakemulley) [reply src/snippets/email-signup.liquid:6]: This is lifted and shifted from MailChimp's own embed code - so if it needs replacing - it's a matter of copy and paste :-)

File: src/styles/modules/email-form.scss (hunk 3 above)
Comment (jakemulley) [reply src/styles/modules/email-form.scss:15]: This extends the `.button` class that uses explicit border-radius parameters. It's not using shorthand to allow preservation of the setting (if it ever changes)

**Review by UsmanAfzal — APPROVED** [review]

**Pull request merged**